src/main/resources/com/nirima/jenkins/plugins/docker/DockerCloud/help-containerCapStr.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The maximum number of containers that this provider is allowed to have running, defaults to blank which disables the limit
```

Comment:
I think @magnayn should review and discuss later plugin usability, because now they are not "user friendly". If value is integer, that it can't be unlimited. zero must be zero - use case for not removing configuration but disabling it.